cartridges/int_styla/cartridge/scripts/StylaMain.js

```diff
@@ -131,247 +128,267 @@ function getContentVersion(config) {
-
-	if (!config.seoDisabled) {
-		config.seoResponse = getSeoContent(config);
-	}
-	
-	config.rootPath = getMagazineRootPath(config.path, config.basePath);
-	
-	return config;
+    var site = Site.getCurrent(),
+        cdnBaseUrl = site.getCustomPreferenceValue('stylaCdnBaseUrl'),
+        cssBaseUrl = site.getCustomPreferenceValue('stylaCssBaseUrl'),
+        version = '',
+        versionResponse,
+        config = {
+            valid: true,
+            path: path,
+            rootPath: null,
+            username: obj.custom.username || '',
+            pipeline: obj.custom.pipeline || '',
+            categoryID: obj.custom.categoryID || '',
+            basePath: obj.custom.basePath || '',
+            seoDisabled: site.getCustomPreferenceValue('stylaSeoDisabled'),
+            seoCachingTTL: site.getCustomPreferenceValue('stylaSeoCachingTTL'),
+            jsLibUrl: cdnBaseUrl.replace('{USER}', obj.custom.username || ''),
+            cssUrl: cssBaseUrl.replace('{USER}', obj.custom.username || '')
+        };
+
+    if (getContentVer) {
+        versionResponse = getContentVersion(config);
+        if (!versionResponse.error && !empty(versionResponse.version)) {
+            version = versionResponse.version;
+        }
+    }
+
+    config.jsLibUrl = config.jsLibUrl.replace('{VERSION}', version);
+    config.cssUrl = config.cssUrl.replace('{VERSION}', version);
+
+    if (!config.seoDisabled) {
+        config.seoResponse = getSeoContent(config);
+    }
+
+    config.rootPath = getMagazineRootPath(config.path, config.basePath);
+
+    return config;
 }
 
 
 /**
  * Check if a given URL @path matches a magazine URL and return the magazine configuration in this case.
- * 
+ *
  * @param path 				The URL path.
  * @param queryString 		The query parameters of the original request.
- * @param getContentVer 
- * 
+ * @param getContentVer
+ *
  * @returns The magazine config values from the site preferences.
  */
 function getMagazineConfiguration(path, getContentVer) {
-	var iter, obj,
-		basePath, idx,
-		localeMatches,
-		locale = request.locale,
-		config = { valid : false };
-
-	if (!empty(path)) {
-
-		//iter = CustomObjectMgr.getAllCustomObjects(CONFIG_CO_TYPE);
-		iter = CustomObjectMgr.queryCustomObjects(CONFIG_CO_TYPE, 'custom.enabled = true', CONFIG_CO_SORT_ORRDER, null);
-		while (iter.hasNext()) {
-			obj = iter.next();
-			// check if this config's basePath is contained in @path 
-			basePath = obj.custom.basePath;
-			idx = path.indexOf(basePath);
-			if (idx != -1) {
+    var iter,
+        iter2,
+        obj,
+        basePath,
+        idx,
+        localeMatches,
+        locale = request.locale,
+        skipHomepageIter = false,
+        config = { valid: false };
+
+    if (!empty(path)) {
+		// iter = CustomObjectMgr.getAllCustomObjects(CONFIG_CO_TYPE);
+        iter = CustomObjectMgr.queryCustomObjects(CONFIG_CO_TYPE, 'custom.enabled = true', CONFIG_CO_SORT_ORRDER, null);

```

Comment:
iter and iter2 use both the same `CustomObjectMgr.queryCustomObjects(CONFIG_CO_TYPE, 'custom.enabled = true', CONFIG_CO_SORT_ORRDER, null)` it seems to me it could be written in one iteration ...